test/blackbox-tests/test-cases/cram-root-dune.t

```diff
@@ -0,0 +1,40 @@
+  $ mkdir -p tests src
+  $ cat > marc.opam << EOF
+  > opam-version: "2.0"
+  > EOF
+  $ cat > dune-project << EOF
+  > (lang dune 3.6)
+  > (cram enable)
+  > EOF
+  $ cat > src/dune << EOF
+  > (executable
+  >   (name marc)
+  >   (public_name marc)
+  >   (package marc)
+  > )
+  > EOF
+  $ echo 'let _ = Printf.printf "I am a dummy test!\n"' > src/marc.ml
+  $ dune build
+  $ dune exec -- marc
+  I am a dummy test!
+
+  $ cat > tests/marc.t << EOF
+  >   $ marc
+  >   I am a dummy test!
+  > EOF
+dun file is at the root
+  $ echo "(cram (deps %{bin:marc}))" > dune
```

Comment:
The issue is this line. By default, the `cram` stanza applies only to the tests in the current directory. If you want it to apply to sub directories,  you need to do `(applies_to :whole_subtree)`.

Reply:
That was my conclusion too. As there is no indication in documentation that it needs to be in the same directory, i thought that it is supported (and dune didn't warn me that something was wrong). Documentation (at least) should be updated with that.